lab_8_llm/main.py

```diff
@@ -4,28 +4,24 @@
 Working with Large Language Models.
 """
 # pylint: disable=too-few-public-methods, undefined-variable, too-many-arguments, super-init-not-called, duplicate-code
-from collections import namedtuple
+#from collections import namedtuple
```

Comment:
Why skip?

Reply:
It is not used in code, so it should be deleted, I just forgor to delete it.